### PR TITLE
Fix of repositories with async operations

### PR DESCRIPTION
**Pull request by DemetriosKoras**
Note: Due to conflict with used interfaces, some methods working with entity framework were left unchanged.

### CODE REVIEW

File: CharlieBackend.Data/Repositories/Impl/AccountRepository.cs [hunk 1]
```diff
@@ -16,29 +16,29 @@ public AccountRepository(ApplicationContext applicationContext)
...
         {
-            return _applicationContext.Accounts
+            return await _applicationContext.Accounts
                 .FirstOrDefaultAsync(account 
                          => account.Email == authenticationModel.Email 
                                 && account.Password == authenticationModel.Password);
         }
 
-		public Task<Account> GetAccountCredentialsById(long id)
+		public async Task<Account> GetAccountCredentialsById(long id)
```
Comment (andrii-developer) [inline CharlieBackend.Data/Repositories/Impl/AccountRepository.cs]: Could you replace these two tabs with spaces and align this method with the others?

**Comment by andrii-developer** [comment]
Would you please also fix the [UnitOfWork.CommitAsync()](https://github.com/ita-social-projects/WhatBackend/blob/53431b2ce265f2e2cc7b59c1e416ac103741c8db/CharlieBackend.Data/Repositories/Impl/UnitOfWork.cs#L170) method?

File: CharlieBackend.Data/Repositories/Impl/AccountRepository.cs (hunk 1 above)
Comment (DemetriosKoras) [reply CharlieBackend.Data/Repositories/Impl/AccountRepository.cs]: Could you please specify which row should I align and relatively to what? I might be blind but it's looking pretty much aligned already.

Comment (andrii-developer) [reply CharlieBackend.Data/Repositories/Impl/AccountRepository.cs]: The row number 27.  It starts with 2 tabs. They should be replaced with 8 spaces.

Comment (DemetriosKoras) [reply CharlieBackend.Data/Repositories/Impl/AccountRepository.cs]: Done, thanks.

**Review by andrii-developer — APPROVED** [review]

**Review by Pasha1968 — APPROVED** [review]

**Review by yuu-ostroglyad — APPROVED** [review]

**Pull request merged**